.rubocop.yml

```diff
@@ -4,6 +4,7 @@ inherit_from:
 AllCops:
   TargetRubyVersion: 1.9
   Exclude:
+    - bundle/**/*
```

Comment:
Sorry, I didn't intend to add it, though without it, trying to develop just using the rake tasks:

```
rake rubocop
<snip>
241 files inspected, 9403 offenses detected

7 errors occurred:
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
An error occurred while Performance/RedundantBlockCall cop was inspecting /Users/glenn.pratt/Code/bundler/bundle/ruby/2.2.0/gems/rack-1.6.4/lib/rack/nulllogger.rb.
Errors are usually caused by RuboCop bugs.
Please, report your problems to RuboCop's issue tracker.
Mention the following information in the issue report:
0.36.0 (using Parser 2.3.0.1, running on ruby 2.2.3 x86_64-darwin14)
RuboCop failed!
```